libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1395,9 +1395,6 @@ function camera:take_picture(instance) end
 ---@class (exact) AP_Camera__camera_state_t_ud
 local AP_Camera__camera_state_t_ud = {}
 
----@return AP_Camera__camera_state_t_ud
-function AP_Camera__camera_state_t() end
-
```

Comment:
This should not have been in the docs generation, the function is not generated on the C++ side. We do not need it because all fields are read only and there are no methods or operators. The docs generator has been fixed to match the C++ generator.